data/hai/hai outfits.txt

```diff
@@ -471,6 +472,17 @@ outfit "Hai Williwaw Cooling"
 	"cooling" 11
 	description "While Hai technology is fairly heat efficient, occasionally they outfit their ships such that they sporadically overheat and shut down. When that happens, they install these."
 
+outfit "Hai Zephyr Cooling"
+	category "Systems"
+	cost 72000
+	thumbnail "outfit/hai zephyr"
+	"mass" 14
+	"outfit space" -14
+	"cooling" 24
+	description `This more advanced cooling system has been developed by the Unfettered with significant help from Korath scientists, who were highly amused with the Williwaw's simultaneous "primitive" design and similarities to a Korath heat shunt.`
```

Comment:
```suggestion
	description `This more advanced cooling system has been developed by the Unfettered with significant help from Korath scientists, who were highly amused that Williwaw had such a "primitive" design yet simultaneously still shared many similarities to a Korath heat shunt.`
```

I'd prefer to adjust it around this way, though :)

Reply:
Removing `advanced` is fine (though you could have used a synonym like `state-of-the-art` or `modern`), but the rest of this sentence reads somewhat awkwardly.

- Calling it `Williwaw` rather than `the Williwaw` is strange and seems to imply personhood rather than clearly labeling it as the proper name of equipment. `the Williwaw cooler` would be even better.
- `yet simultaneously still` sounds weird. I would remove `still`.
- You're switching tense mid-sentence.

I think my suggestion above is more suitable.